scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -94,7 +94,7 @@ fi
 if [ "$os" == "auto" ] ; then
     os=$( . /etc/os-release ; echo "${ID}${VERSION_ID}" )
     case $os in
-        centos7|ubuntu18.04|ubuntu20.04) [ -z "$print" ] && echo "Detected OS: ${os}" ;;
+        centos7|rhel8.2|ubuntu18.04|ubuntu20.04) [ -z "$print" ] && echo "Detected OS: ${os}" ;;
```

Comment:
Commentary with rhel8.* is valid, but I don't sure about centos8

Reply:
I just thought that it would be great to have ability to check rhel8 branch without registration:
```
# yum install -y gstreamer1
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 0:00:39 ago on Fri Feb 12 09:25:36 2021.
No match for argument: gstreamer1
Error: Unable to find a match: gstreamer1
```